chapters/scoping.tex

```diff
@@ -123,6 +128,11 @@ \subsection{Composite Name Lookup}\label{composite-name-lookup}
 \end{lstlisting}
 \end{example}
 
+\begin{nonnormative}
+Note that an \lstinline!operator! class may only contain declarations of functions and thus fulfills the requirements for a package (see \cref{specialized-classes}).
+In practice, the non-deprecated rules imply that we can call \lstinline!Complex.'-'.negate! and \lstinline!Complex.'+'! for the example in \cref{example-of-overloading-for-complex-numbers}.
+This requires that \lstinline!operator '-'! and \lstinline!operator function '+'! are declared as \lstinline!encapsulated! as in the example.
+\end{nonnormative}
```

Comment:
Restoring the double empty line spacing befor the `\subsection` to make it easier to find sectioning when scrolling through the source:
```suggestion
\end{nonnormative}

```